site/docs/master/velero-install.md

```diff
@@ -0,0 +1,46 @@
+# Velero Install CLI
+
+This document serves as a guide to using the `velero install` CLI command to install `velero` server components into your kubernetes cluster.
+
+_NOTE_: `velero install` will, by default, use the version information to determine the version of of the server compoents to deploy. This behavior may be overridden by using the `--image` flag. Refer to [Building Server Component Container Images For Server Component][1].
+
+## Usage
+
+This section explains some of the basic flags supported by the `velero install` CLI command. For a complete explaination of the flags, please run `velero install --help`
+
+```(bash)
+velero install \
+    --plugins <PLUGIN_CONTAINER_IMAGE [PLUGIN_CONTAINER_IMAGE]>
+    --provider <YOUR_PROVIDER> \
+    --bucket <YOUR_BUCKET> \
+    --secret-file <PATH_TO_FILE> \
+    --velero-pod-cpu-request <CPU_REQUEST> \
+    --velero-pod-mem-request <MEMORY_REQUEST> \
+    --velero-pod-cpu-limit <CPU_LIMIT> \
+    --velero-pod-mem-limit <MEMORY_LIMIT> \
+    [--use-restic] \
+    [--restic-pod-cpu-request <CPU_REQUEST>] \
+    [--restic-pod-mem-request <MEMORY_REQUEST>] \
+    [--restic-pod-cpu-limit <CPU_LIMIT>] \
+    [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+In the about usage explanation:
+
+1. The values for the resource requests and limits flags follow the same format as [Kubernetes resource requirements][5]
+1. For plugin container images, please refer to our [supported providers][2] page.
+
+## Examples
+
+This section provides examples that serves as a starting point for more customized installations.
+
+```(bash)
```

Comment:
If you use `bash` or `console` after the backticks, GitHub will do syntax highlighting, but it won't apply to Hugo (that I'm aware of)

Reply:
It rendered pretty well. https://deploy-preview-2155--velero.netlify.com/docs/master/velero-install/#usage
WDYT?

Reply:
Yeah, it will render with the standard monospace code block, but without the parentheses, it should also additional highlighting.

Reply:
OK.. let's use that then!

Reply:
Looks like the parens on L35 can still be removed.

Reply:
@ashish-amarnath looks like this comment still needs to be addressed (remove parens)